Incorrect substitution in error message for `B027`

<!--
Thank you for taking the time to report an issue! We're glad to have you involved with Ruff.

If you're filing a bug report, please consider including the following information:

- A minimal code snippet that reproduces the bug.
- The command you invoked (e.g., `ruff /path/to/file.py --fix`), ideally including the `--isolated` flag.
- The current Ruff settings (any relevant sections from your `pyproject.toml`).
- The current Ruff version (`ruff --version`).
-->


For the following minimal reproducer:

```python
import abc

class Class(abc.ABC):
    def method(self):
        pass
```

I get the following:

```console
❯ ruff --version
ruff 0.0.238

❯ ruff --select B027 bug.py
bug.py:4:5: B027 `Class` is an empty method in an abstract base class, but has no abstract decorator
Found 1 error.
```

I'd expect either `Class.method()` or `method()` instead of `Class`.

Not sure whether `"... no abstract decorator"` would be better as `"... no abstractmethod decorator"` or `"... no @abstractmethod decorator"`?

--- a/src/rules/flake8_bugbear/rules/abstract_base_class.rs
+++ b/src/rules/flake8_bugbear/rules/abstract_base_class.rs
@@ -62,16 +62,19 @@ pub fn abstract_base_class(
             continue;
         }
 
-        let (StmtKind::FunctionDef {
+        let (
+            StmtKind::FunctionDef {
                 decorator_list,
                 body,
+                name: method_name,
+                ..
+            } | StmtKind::AsyncFunctionDef {
+                decorator_list,
+                body,
+                name: method_name,
                 ..
             }
-            | StmtKind::AsyncFunctionDef {
-                decorator_list,
-                body,
-                ..
-            }) = &stmt.node else {
+        ) = &stmt.node else {
             continue;
         };
 
@@ -89,7 +92,7 @@ pub fn abstract_base_class(
         if !has_abstract_decorator && is_empty_body(body) && !is_overload(checker, decorator_list) {
             checker.diagnostics.push(Diagnostic::new(
                 violations::EmptyMethodWithoutAbstractDecorator {
-                    name: name.to_string(),
+                    name: format!("{name}.{method_name}"),
                 },
                 Range::from_located(stmt),
             ));

